src/main/controlMod.F90

```diff
@@ -363,6 +365,10 @@ subroutine control_init( )
           call endrun(msg=' ERROR: expecting n_dom_pfts between 0 and 14 where 0 is the default value that tells the model to do nothing ' // &
                errMsg(sourcefile, __LINE__))
        end if
+       if (n_dom_landunits < 0) then
```

Comment:
Should it also check for the maximum number of possible landunits? I think it should...

Reply:
Done.